src/control/controlindicatortimer.cpp

```diff
@@ -0,0 +1,68 @@
+#include "control/controlindicatortimer.h"
+
+#include "control/controlobject.h"
+#include "moc_controlindicatortimer.cpp"
+#include "util/math.h"
+
+namespace mixxx {
+
+ControlIndicatorTimer::ControlIndicatorTimer(QObject* pParent)
+        : QObject(pParent),
+          m_pCOIndicator250millis(std::make_unique<ControlObject>(
+                  ConfigKey("[Master]", "indicator_250millis"))),
+          m_pCOIndicator500millis(std::make_unique<ControlObject>(
+                  ConfigKey("[Master]", "indicator_500millis"))),
+          m_toggleIndicator500millisOnNextTimeout(true),
+          m_nextSwitchTime(0.0),
+          m_pCPGuiTick50ms(nullptr) {
+    m_pCOIndicator250millis->setReadOnly();
+    m_pCOIndicator500millis->setReadOnly();
+    connect(&m_timer, &QTimer::timeout, this, &ControlIndicatorTimer::slotTimeout);
+    m_timer.start(250);
+}
+
+void ControlIndicatorTimer::slotTimeout() {
```

Comment:
```suggestion
void ControlIndicatorTimer::slotQTimerTimeout() {
```
to be clear this is unrelated to the legacy vsync code below

Reply:
It's still called by the legacy code below. I separated the code in both the header file and the cpp file, so I think there shouldn't be much confusion.

Reply:
okay